docs/website/docs/getting-started/tflite.md

```diff
@@ -0,0 +1,54 @@
+# TFLite Integration
+
+IREE supports coming and running TensorFlow-Lite programs stored as [TFLite
+flatbuffers](https://www.tensorflow.org/lite/guide). These files can be
+imported into an IREE compatible format then compiled to a series of backends.
```

Comment:
style nit: I prefer to not break a link title across multiple lines

```suggestion
IREE supports coming and running TensorFlow-Lite programs stored as
[TFLite flatbuffers](https://www.tensorflow.org/lite/guide). These files can be
imported into an IREE compatible format then compiled to a series of backends.
```

Reply:
Done.